docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>(The output seems stale. I usually try to remember to factory reset the notebook and run end to end before submitting.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I'm pretty sure that's what I did. Will check, but I think it's because I switched from colab to local with vscode (hence packets stick around).</p>